frontend/public/style/_overrides.scss

```diff
@@ -189,3 +189,11 @@ tags-input .autocomplete .suggestion-item em {
 .modal-footer .alert {
   text-align: left;
 }
+
+// FIXME: Pass as `style` prop to `List` once this is resolved (https://github.com/bvaughn/react-virtualized/issues/876)
+.ReactVirtualized__Grid.ReactVirtualized__List {
```

Comment:
@kans Is this an acceptable solution for now? I played around with `ReactDOM.createPortal()`, but it requires polluting our `props` with X and Y positioning (see https://codepen.io/jscottsmith/pen/xYaawK). We can open a PR upstream that lets us override it in the `style` prop passed to `List` if needed.